pkg/features/kube_features.go

```diff
@@ -749,6 +749,16 @@ const (
 	//
 	// Enables the image volume source.
 	ImageVolume featuregate.Feature = "ImageVolume"
+
+	// owner: @derekwaynecarr, @zhifei92
```

Comment:
```suggestion
	// owner: @zhifei92
```

Why Derek was a co-owner?

Reply:
To be honest, I'm not quite clear what the owner means here. I thought it referred to the designer and author.

Reply:
just keep your alias.